examples/compose/docker-compose-with-web.yml

```diff
@@ -0,0 +1,57 @@
+version: '2.1'
+
+services:
+  redis:
+    image: autopilotpattern/redis:3.2.8-r1.0.0
+    mem_limit: 512m
+    restart: always
+    expose:
+      - 6379
+      - 26379
+    environment:
+      - CONSUL=consul
+    links:
+      - consul:consul
+    ports:
```

Comment:
Should we add a comment here about how this and most of the other `ports` declarations should not be open for production, at least on Triton?

See also https://github.com/autopilotpattern/wordpress/issues/44